src/components/work/overview/bookmarkDropdown/BookmarkDropDown.module.css

```diff
@@ -3,38 +3,73 @@
 
 .dropdownitem {
   display: block;
-  border-bottom: 1px solid var(--mine-shaft);
-  padding: var(--pt1) var(--pt2) var(--pt1) var(--pt2);
+  border-bottom: 1px solid var(--gray);
+  padding: var(--pt1) var(--pt05) var(--pt1) var(--pt2);
   text-decoration: none;
   color: var(--mine-shaft);
   text-align: inherit;
   white-space: nowrap;
   cursor: pointer;
 }
 
+.dropdownitem i svg {
+  display: none;
+}
+
+.bookmark {
+  margin-left: var(--pt2);
+}
+
 .active {
   color: white;
-  background-color: green;
+  background-color: var(--blue);
+}
+
+.dropdownitem.active i svg {
+  display: block;
+}
+
+.active i svg path {
```

Comment:
Nu har vi den opsætning vi nu engang har. Man kan undgå denne lidt grimme css selector hvis man opretter svg'en som et react component, og sætter stroke og path til "currentColor" - så kan farven styres af css "color". 

Da vi ikke har gjort dette før, så lad det ligge. 

Du kunne evt spare "i" selector ved at give Icon dit classname istedet for at styre det med en parent.